extensions/cornerstone-dicom-sr/src/getSopClassHandlerModule.ts

```diff
@@ -50,15 +64,35 @@ const RELATIONSHIP_TYPE = {
 
 const CORNERSTONE_FREETEXT_CODE_VALUE = 'CORNERSTONEFREETEXT';
 
+/**
+ * Adds instances to the DICOM SR series, rather than creating a new
+ * series, so that as SR's are saved, they append to the series, and the
+ * key image display set gets updated as well, containing just the new series.
+ * @param instances is a list of instances from THIS series that are not
+ *     in this DICOM SR Display Set already.
+ */
+function addInstances(
+  instances: InstanceMetadata[],
+  displaySetService: DisplaySetService
+) {
+  this.instances.push(...instances);
+  utils.sortStudyInstances(this.instances);
+  this.instance = this.instances[this.instances.length - 1];
```

Comment:
Why the last instance and not the first?

Reply:
The last instance is the newest one, so is the one most interesting.  Eventually it should have the ability to choose which one gets loaded, and to navigate among them.

Reply:
> The last instance is the newest one, so is the one most interesting. Eventually it should have the ability to choose which one gets loaded, and to navigate among them.

Consider adding what you just mentioned as a comment in the code.